<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
  <head>

    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=0, shrink-to-fit=no">
    <meta name="description" content="">
    
    <meta name="author" content="">

    <title>RBP-Site</title>


    <!-- Bootstrap core CSS -->
    <link href="vendor/bootstrap/css/bootstrap.min.css" rel="stylesheet">

    <!-- Custom fonts for this template -->
    <link href="vendor/fontawesome-free/css/all.min.css" rel="stylesheet">
    <link href="vendor/simple-line-icons/css/simple-line-icons.css" rel="stylesheet" type="text/css">
    <!<link href="https://fonts.googleapis.com/css?family=Lato:300,400,700,300italic,400italic,700italic" rel="stylesheet" type="text/css">

    <!-- Custom styles for this template -->
    <link href="css/landing-page.min.css" rel="stylesheet">
<style>
   #page-wrap {
   margin-left:auto;
  margin-right:auto;
    width:960px;
}




#myImg {
  border-radius: 5px;
  cursor: pointer;
  transition: 0.3s;
}

#myImg:hover {opacity: 0.7;}

/* The Modal (background) */
.modal {
  display: none; /* Hidden by default */
  position: fixed; /* Stay in place */
  z-index: 1; /* Sit on top */
  padding-top: 100px; /* Location of the box */
  left: 0;
  top: 0;
  width: 100%; /* Full width */
  height: 100%; /* Full height */
  overflow: auto; /* Enable scroll if needed */
  background-color: rgb(0,0,0); /* Fallback color */
  background-color: rgba(0,0,0,0.9); /* Black w/ opacity */
}

/* Modal Content (image) */
.modal-content {
  margin: auto;
  display: block;
  width: 80%;
  max-width: 700px;
}

/* Caption of Modal Image */
#caption {
  margin: auto;
  display: block;
  width: 80%;
  max-width: 700px;
  text-align: center;
  color: #ccc;
  padding: 10px 0;
  height: 150px;
}

/* Add Animation */
.modal-content, #caption {  
  -webkit-animation-name: zoom;
  -webkit-animation-duration: 0.6s;
  animation-name: zoom;
  animation-duration: 0.6s;
}

@-webkit-keyframes zoom {
  from {-webkit-transform:scale(0)} 
  to {-webkit-transform:scale(1)}
}

@keyframes zoom {
  from {transform:scale(0)} 
  to {transform:scale(1)}
}

/* The Close Button */
.close {
  position: absolute;
  top: 15px;
  right: 35px;
  color: #f1f1f1;
  font-size: 40px;
  font-weight: bold;
  transition: 0.3s;
}

.close:hover,
.close:focus {
  color: #bbb;
  text-decoration: none;
  cursor: pointer;
}

/* 100% Image Width on Smaller Screens */
@media only screen and (max-width: 700px){
  .modal-content {
    width: 100%;
  }

</style>
    
</head>
 <body>
<div id="page-wrap">
<!-- Navigation -->

    <nav class="navbar fixed-top navbar navbar-light bg-light justify-content-between ">
      <a class="navbar-brand" href="http://scbb.ihbt.res.in/" style="max-width: 5%;">
    <img src="img/IHBT.png" class="img-fluid">
<div>
<a class="btn btn-danger" href="#">Home</a>
        <a class="btn btn-danger" href="http://scbb.ihbt.res.in/">About Us</a>
        <div class="btn-group">
  <button type="button" class="btn btn-danger">Plots</button>
  <button type="button" class="btn btn-danger dropdown-toggle px-3" data-toggle="dropdown" aria-haspopup="true"
    aria-expanded="false">
    <span class="sr-only">Toggle Dropdown</span>
  </button>
  <div class="dropdown-menu scrollable-menu" role="menu" >
    <a class="dropdown-item" href="responsive_image_website_27feb/zoombar.html"><b>Number of RBPs on each miRNA</b></a>
    <a class="dropdown-item" href="responsive_image_website_27feb/scattereach_RBP_miRNA.html"><b>miRNA and their associated RBPs<b></a>
    <a class="dropdown-item" href="responsive_image_website_27feb/bubble/index.html"><b>Position of RBPs on each miRNA<b></a>
    <a class="dropdown-item" href="responsive_image_website_27feb/corellelogram_plot/index.html"><b>Corellogram plot<b></a>
  </div>
</div>
        <a class="btn btn-danger" href="gallery.html">Gallery</a>
        <a class="btn btn-danger" href="#team-page">Our Team</a>
        <a class="btn btn-danger" href="contact.html">Contact-Us</a>
</div>
   <a class="navbar-brand" href="https://www.csir.res.in/"><img src="img/csir_logo_1_214x236.png"class="float-right" alt="logo" style="max-width:60%;"></a>
  </div>
      </div>
    </nav>
<!-- Masthead -->
    <header class="masthead text-white text-center">
      <div class="overlay"></div>
      <div class="container">
        <div class="row">
          <div class="col-xl-9 mx-auto">
          <h1 class="mb-5"><em> RBP-miRNA Crosstalking</em></h1>
             <h1>RBP-miRNA Crosstalking</h1>
             <p>There are always group of RBPs for regulation of miRNA</p>
          </div>
          
      </div>
    </header>
<nav class="navbar navbar-dark bg-primary">
  <!-- Navbar content -->
<!--<button class="navbar-toggler" type="button" data-toggle="#" data-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
            <span class="navbar-toggler-icon"></span>
        </button>
        <div class="collapse navbar-collapse" id="navbarSupportedContent">
            <ul class="navbar-nav mr-auto">
                <li class="nav-item active">
                    <a class="nav-link" href="#">Home <span class="sr-only">(current)</span></a>
                </li>
                <li class="nav-item">
                    <a class="nav-link" href="http://scbb.ihbt.res.in/">About Us</a>
                </li>
                <li class="nav-item">
                    <a class="nav-link" href="#">Plots</a>
                </li>
                <li class="nav-item">
                    <a class="nav-link" href="gallery.html">Gallery</a>
                </li>
                 <li class="nav-item">
                    <a class="nav-link" href="#">OurTeam</a>
                </li>
                <li class="nav-item">
                    <a class="nav-link" href="contact.html">Contact US</a>
                </li
            </ul>
        </div>
    </nav>
</nav>-->
   <!--Accordion wrapper-->
<div class="accordion md-accordion" id="accordionEx" role="tablist" aria-multiselectable="true">

  <!-- Accordion card -->
  <div class="card">

    <!-- Card header -->
    <div class="card-header" role="tab" id="headingOne1">
      <a data-toggle="collapse" data-parent="#accordionEx" href="#collapseOne1" aria-expanded="true"
        aria-controls="collapseOne1">
        <h5 class="mb-0">
          ABSTRACT <i class="fa fa-angle-down rotate-icon"></i>
        </h5>
      </a>
    </div>

    <!-- Card body -->
    <div id="collapseOne1" class="collapse show" role="tabpanel" aria-labelledby="headingOne1" data-parent="#accordionEx">
    <div class="card-body" style= " border: 3px solid green; text-align:justify"><br>
  Until now, typically gene regulation have been identified which include:-
 (I) Rate of Gene transcription and mRNA decay determining the steady state level of transcript.
 (II) Transcription along with translation rate determine the amount of protein produced.
 (III) Transcription is controlled by DNA Binding Protein (DBP), post-transcriptional regulation such as m-RNA decay and translation is mediated by RNA Binding Protein (RBP) which form ribonucleoprotein complex with transcripts.
  Gene expression steps are highly co-ordinated and interdependent.
 Different techniques which are present to find global RNA-protein interaction network are not able to uncover this task because of many reason which are given below:
 (a) Available interaction data cover a quite small fraction of known RBPs.
 (b) Experimentally determined interaction are often noisy and cell line dependent. Even for RBP with experimentally determined interactomes, information are not fully accurate. Due to dependency of those techniques on expression level and cell lines, some kind of interaction might be missed.
 (c) These techniques individuate the binding sites of single RBP of interest in each experiment. Exception is gPar-CLIP. That determine mRNA-bound proteome and its global occupancy profile.
 gPar-CLIP does not allow match binding sites with specific RBPs, thus it does not provide information how multiple RBP target same mRNAs. Similarly, PTR combiner extract cluster of mRNA co-regulator from interaction data. It employ a pattern set mining techniques based on boolean matrix factorization to extract cluster of co-regulator RBPs. It also provide biological analysis of extracted cluster that might suggest functional characterization of set of mRNAs targeted by cluster of regulators and of the binding dynamics of different RBPs belong to the some cluster.
 Map CLIP binding site of 60 RBP on human genome to construct RBP-RNA network. Examine correlation of each RBPs expression at both transcripts and protein level with target RNA to see how these correlation patterns changes.Using modeling techniques like multivariate regression modeling stepwise linear regression and elastic net to analyzed deflect factor impacting change in expression pattern through two level Analysis. RBP at both proteomic and transcriptomic level play an important role in coordinating expression change of target RNA. This is defined by various factors govering functions of these RBPs.  
 To understand how RBPs are associated with target, we correlate expression of one of protein coding transcript of each RBP with expression of target transcripts across 18 human tissues and compared it with control set of transcripts. 
      </div>
    </div>

  </div>
  <!-- Accordion card -->

  <!-- Accordion card -->
  <div class="card">

    <!-- Card header -->
    <div class="card-header" role="tab" id="headingTwo2">
      <a class="collapsed" data-toggle="collapse" data-parent="#accordionEx" href="#collapseTwo2"
        aria-expanded="false" aria-controls="collapseTwo2">
        <h5 class="mb-0">
          INTRODUCTION <i class="fa fa-angle-down rotate-icon"></i>
        </h5>
      </a>
    </div>

    <!-- Card body -->
    <div id="collapseTwo2" class="collapse" role="tabpanel" aria-labelledby="headingTwo2" data-parent="#accordionEx">
     <div class="card-body">
      <div class="container-fluid">
    <div class="row">
    <div class="col" style=" border: 3px solid green; text-align: justify"><br>
      MicroRNAs (miRNAs) are ~21 to ~24 nucleotide  RNA molecules, play important regulators of post transcriptional gene expression and have been  involved  in different cellular pathways (Bushati and Cohen, 2007). The biogenesis of miRNAs is a complex process primarily  dependent  by RNA binding proteins (RBPs). The canonical miRNA biogenesis start with Precursor miRNAs (Pre-miRNAs) hairpins, which are embedded within exonic and intronic regions of host protein-coding and non-coding primary miRNAs (Pri-miRNAs) transcripts, are processed  by microprocessor complex. microprocessor complex  comprehend with  RNase  III DROSHA and the double-stranded (ds) RBP DGCR8 (Han et al., 2004; Gregory et al., 2004; Denli et al., 2004).  DROSHA cleaves off flanking single stranded sequences and generates a miRNA precursor  that is transported to the cytoplasm from nucleus by exportin-5 protein, which is also a RBP mostly helps in transportation (Kim et al., 2009).  In the cytoplasm, the RNase III DICER1 binds loop region of  pre-miRNAs and cleaved it to produce the mature 3p/5p miRNA duplex . DICER1  is a  RNase III enzyme which contains an ATPase/RNA helicase, a PAZ domain  (Yan et al.,2003), two catalytic RNase III domains, and a C-terminal dsRBD (Lau et al.,2012).  With the help of the dsRBPs  TRBP or PACT, either 3p or 5p strand is selected and loaded into the RNA induced silencing complex(RISC), where it directly interacts with a member of the Argonaute protein family (Dueck and Meister, 2014; Kim et al.,2009).  Argonautes (AGO 1-4) are bi-lobed proteins, they consist of an N-terminal , a PAZ domain (similar to the PAZ domain found in Dicer), a middle (MID) domain and a PIWI domain (Song et al.,2004; Elkayam et al., 2012; Schirle et al. , 2012). Ago2 (one of the four mammalian Argonautes) is the only Ago that possesses endonucleolytic activity and can initiate the cleavage of the passenger strand of an extensively base-paired small RNA duplex or the targeted mRNA if it is perfectly complementary to the Ago2 bound small RNA (Song et al.,2004; Matranga et al.,2005; Yuen et al.,2005; Parker et al.,2004; Wang et al., 2008). <div><img id="myImg" src="img/miRNA_bognesis_08.jpeg" alt="mirna_biogenesis" style="width:100%;max-width:auto;"><br><h6><b>figure 1:<span></span></b>miRNA Biogenesis Process</h6>

<!-- The Modal -->
<div id="myModal_0" class="modal">
  <span class="close">&times;</span>
  <img class="modal-content" id="img0">
  <div id="caption"></div>
</div>
</div>

<script>
// Get the modal
var modal = document.getElementById('myModal_0');

// Get the image and insert it inside the modal - use its "alt" text as a caption
var img = document.getElementById('myImg_0');
var modalImg = document.getElementById("img_0");
var captionText = document.getElementById("caption");
img.onclick = function(){
  modal.style.display = "block";
  modalImg.src = this.src;
  captionText.innerHTML = this.alt;
}

// Get the <span> element that closes the modal
var span = document.getElementsByClassName("close")[0];

// When the user clicks on <span> (x), close the modal
span.onclick = function() { 
  modal.style.display = "none";
}
</script>
     <br>An increasing amount of evidence also suggests that cleavage competent Argonautes also play a role in the processing of a pri-miRNA in C. elegans and in the Dicer dependent and independent maturation of pre-miRNAs in mammals and worms (Zisoulis et al.,2012; Diederichs et al.,2007; Bouasker et al.,2012; Cifuentes et al.2010).RBPs could also regulate miRNA biogenesis by affecting the expression or stability of the canonical proteins of the miRNA pathway. Cells have the ability to utilize a wide spectrum of mechanisms to produce functional miRNAs. Although the afore mentioned canonical pathway accounts for the production of most miRNAs, it has also been shown that there exist alternative pathways for miRNA biogenesis, which bypass a part of the biogenesis steps mentioned above.
    </div>
    <div class="col" style=" border: 3px solid green;text-align:justify"><br>
  These miRNAs are therefore termed non-canonical miRNAs. The processing and action of miRNAs are extensively regulated by auxiliary factors to ensure cell/tissue specific functions or adequate response to environmental and cellular stimuli. One of the largest groups of proteins that influence the miRNA pathways are RNA-binding proteins (RBPs).  Similar to protein-coding genes, non-coding pri-miRNAs expression is subject to post-transcriptional control. Yet, only a small fraction (1%–5%) of RBPs have been demonstrated to control the various stages of miRNA processing in an endogenous context, though till now there are 5000 RBPs have been identified in human (Ghosh et al.,2018) . Jha et al,  (2011), while working  to understand the process of post-transcriptional regulation, processing and handling of miRNA forms, using expression data and scanning for some known RBP motifs they  found that miRNAs have potential very high relationship with RBPs, and several RBPs could be involved in miRNA formation and translocation besides well known Drosha/Dicer/Argonaute proteins. While the RBPs responsible for the general processing of miRNAs namely, DROSHA and DGCR8 are very well characterized, where other RBPs can also play a more important  role in miRNA biogenesis to affect a specific subset of miRNAs. Now, very recently due to technological advancement some NGS methods have been developed where transcriptome wide RBP interactome could be discovered by using  a variety of cross-linking immuno-precipitations (CLIP-seq). These methods include PAR-CLIP/HITS-CLIP/iCLIP/eCLIP/ir-CLIP CLEAR-CLIP/miCLIP/PAR-iCLIP/PAR-CLIP-MeRIP etc. However, despite of their high-throughput nature, these methods are specifically designed for any given RBP and don't provide information about all RBPs. Although a number of RNA-binding proteins have been implicated in miRNA biogenesis, a comprehensive picture of the dynamic interactome of pri-/pre-miRNAs is still missing.  Therefore, using the above mentioned experiments one can get enough data to model the interactions of any given number of RBPs.   To gain a full understanding of miRNAs processing there is  a need to identify and classify additional regulators of miRNA biogenesis. Thus far, systematic and comprehensive approaches to identify RBP:miRNA interactions have been rare and limited in their utility. The present study has realized these developments and  proposes a solution to decipher miRNAs and their RBP association in miRNA biogenesis . In this paper we have presented the regulatory models for RBP dependent miRNA biogenesis models and characterized miRNAs based on their RBP association considering multi-experimental conditions.  which include different  experimental  conditions (disease/control) and different tissue types of human.<br><br><br><img class= "center" id="myImg1_01" src="img/Introduction_2.png" alt=trend" style="width:100%;max-width:auto; border:solid;"><br><br><br>
       <!-- The Modal -->
<div id="myModal_2" class="modal">
  <span class="close">&times;</span>
  <img class="modal-content" id="img01_01">
  <div id="caption"></div>
</div>
    </div>
  </div>

    </div>
   
    </div>
    

  </div>
  <!-- Accordion card -->

  <!-- Accordion card -->
  <div class="card">

    <!-- Card header -->
    <div class="card-header" role="tab" id="headingThree3">
      <a class="collapsed" data-toggle="collapse" data-parent="#accordionEx" href="#collapseThree3"
        aria-expanded="false" aria-controls="collapseThree3">
        <h5 class="mb-0">

         MATERIAL and METHOD <i class="fa fa-angle-down rotate-icon"></i>
        </h5>
      </a>
    </div>

    <!-- Card body -->
    <div id="collapseThree3" class="collapse" role="tabpanel" aria-labelledby="headingThree3" data-parent="#accordionEx">
    <div class="card-body">
    <div class="container-fluid">
    <div class="row">
    <div class="col" style=" border: 3px solid green;text-align: justify"><br>
       Data for miRNA-seq and RNA-seq based high throughput studies were collected from Gene Expression Omnibus (GEO) and Sequence Read Archive (SRA) for 20 experiments which included 47 different experimental conditions. Complete list of various experimental conditions (for both RNA-seq and miRNA-seq) and source of data along with detail experimental description are  available in Supplementary Table-1. <br><embed src="table/Table-1.pdf" width="650px" height="250px"/><br> The miRNA sequences of human (1,881 precursor miRNA; 2,588 mature miRNA) are collected from the the mirBase database version 21 (Kozomara et al., 2010).  1230 CLIP-seq experiments   high confidence processed data for  155 RBPs are collected from ENCODE and GEO  database. These datasets based on  different types of CLIP-seq experiments such as: PAR-CLIP, PAR-CLIP-MeRIP, HITS-CLIP, eCLIP, iCLIP, CLEAR-CLIP and irCLIP techniques which include different experimental conditions, the detail description of datasets are presented in Table-2.<br><embed src="table/Table-2.pdf" width="650px" height="250px"/><br>
       RNA-seq and miRNA seq data were filtered using filteR (Gahlan et al.,2012) and trimometric (Bolgar et al., 2014).   For mapping RNA-seq reads across the human genome (hg38) Seqmap  has been used which is based on  Bowtie platform. The alignment result was saved in BAM format for quantification. RSEQ tool (Jiang et al.,2009) has been implemented for quantification of  gene expression  from BAM files. For expression analysis of Pre-miRNAs  same RNA-seq pipeline  was   carry out, but here RNA-seq reads were  mapped over pre-miRNA sequences collected from mirBase and the alignment result are store in a BAM file. RSEQ  tool was  used for quantification of Pre-miRNA expression   from RNA-seq data  . MirDeep2 tool (Marc et al.,2012) was used for expression analysis of  mature miRNA from miRNA-seq data. For mapping  miRNA-seq reads  over  mature miRNAs collected from mirBase mapper.pl script has been used. Quantifier.pl script has been executed for quantification of mature miRNA expression  from the saved BAM files. The details pipeline followed for expression analysis in  Pre-miRNA, RBPs and mature miRNA  are presented in Fig 2. Out of 155 RBPs 46 RBPs are collected from ENCODE database which include 222 experiments . All bed files of 46 RBPs are collected from ENCODE , which does not require further processing. Besides these 46 RBPs, CLIP-seq data for 109 RBPs are collected from GEO which included 1008 experiments . All these raw reads are processed following a standard  protocol and the detail pipeline presented in fig.2. <br><img class= "center" id="myImg_2" src="img/image2.png" alt="expression analysis" style="width:100%;max-width:600px; border:solid;"><br>
       <!-- The Modal -->
<div id="myModal_2" class="modal">
  <span class="close">&times;</span>
  <img class="modal-content" id="img02">
  <div id="caption"></div>
</div>

<script>
// Get the modal
var modal = document.getElementById('myModal_2');

// Get the image and insert it inside the modal - use its "alt" text as a caption
var img = document.getElementById('myImg_2');
var modalImg = document.getElementById("img02");
var captionText = document.getElementById("caption");
img.onclick = function(){
  modal.style.display = "block";
  modalImg.src = this.src;
  captionText.innerHTML = this.alt;
}

// Get the <span> element that closes the modal
var span = document.getElementsByClassName("close")[1];

// When the user clicks on <span> (x), close the modal
span.onclick = function() { 
  modal.style.display = "none";
}
</script>
       
       Till now There is   limited   data available regarding primary miRNA , also very few tools are available for Pri-miRNA prediction . So  pre-miRNA collected from mirBase database are extened upto 1kb flanking region on both side, which we considered  as Pri-miRNA in our study. To find out the possible binding site of different RBPs on pri-miRNA and pre-miRNA, Bedtool has been implemented to intersect coordinates of RBP binding sites with Pri-miRNA and Pre-miRNA using bedfiles as input for 46 RBPs collected from ENCODE database. 
       </div>
       <div class="col" style=" border: 3px solid green; text-align:justify"><br>
        The processed CLIP-seq sequences for 109 RBPs collected from GEO  are  mapped on pri-miRNA and Pre-miRNA using Bowtie  to get possible binding site of these 109 RBPs. After getting the result of RBP binding sites on  Pri-miRNA and Pre-miRNA, these binding sites are clustered based on each RBPs, Pri-miRNAs and Pre-miRNAs to get a clear statistics on binding sites of each RBP on different miRNAs and number of different RBPs bind on a particular miRNA. A very important observation about RBPs is that they follow regulon model of regulation where genes disposing towards some common functions have been found commonly regulated by some common RBPs (Keene, 2007). To get the regulon of each RBP, initially possible partners of those 155 RBPs have been collected  from String database (Szklarczyk, D. et al.) which include  RBPs and genes. There are total 1169 (804 RBPs + 365 genes) possible partners of these 155 RBPs are collected  and we don't have the  CLIP-seq data for other RBPs besides these 155 RBPs. so there is a need to establish a general relationship between the binding sites and expression (up/down) of these RBPs. To establish a general view CLIP-seq data and RNA-seq data in same experimental conditions collected for 73 RBPs and the detail description of these data are presented in Table-3. Again these CLIP-seq data are processed following the protocol described  previously and are mapped on human genome (hg38) to find possible binding sites for that particular RBP in human genome  conditionwise.  Similarly RNA-seq expression analysis has been performed to quantify  the expression of  RBP in that particular condition.  Correlation analysis  has been carry out to check the association between binding sites and expression for different RBPs in different experimental conditions. <br><img class= "center" id="myImg_3" src="img/image3.jpeg" alt="expression of rbp" style="width:100%;max-width:600px; border:solid;"><br>
         <!-- The Modal -->
	<div id="myModal_3" class="modal">
  <span class="close">&times;</span>
  <img class="modal-content" id="img03">
  <div id="caption"></div>
</div>

<script>
// Get the modal
var modal = document.getElementById('myModal_3');

// Get the image and insert it inside the modal - use its "alt" text as a caption
var img = document.getElementById('myImg_3');
var modalImg = document.getElementById("img03");
var captionText = document.getElementById("caption");
img.onclick = function(){
  modal.style.display = "block";
  modalImg.src = this.src;
  captionText.innerHTML = this.alt;
}

// Get the <span> element that closes the modal
var span = document.getElementsByClassName("close")[2];

// When the user clicks on <span> (x), close the modal
span.onclick = function() { 
  modal.style.display = "none";
}
</script>
         Due to few number of samples in our  study, we had decided a  correlation 0.8  for strong association. Those  RBPs showed distinctive behavior in correlation i.e. in some conditions correlation greater than  0.8 where as  in  other conditions it is less than 0.8 . The difference in correlation of same RBP in different conditions might be due to interference of other gene or RBPs ,which impact the expression and binding  of that particular RBP.  To address this distinctive behavior  of correlation in different conditions of same RBP  and to figure out possible interaction (synergistic/antagonistic)  with other RBPs or genes  a co-expression network analysis has been carried out  using a in-house prepared R-script. To perform the co-expression analysis, initially  the possible partners were collected for each RBPs from the string database. Again based on the expression data of  RBP and their partners in particular condition, their interaction (both positive/negative ) has been evaluated from co-expression network. To study the involvement of possible RBPs and their regulons in miRNA biogenesis a Bayesian Network Anlysis (BNA) has been performed  based on expression data of  miRNAs, RBPs and possible partners of RBPs collected from string database. BNA has been performed for all 47 experimental conditions and the input  consisting of 1881 Pre-miRNAs, 2588 mature miRNAs , 155 RBPs and 1169 genes . But in each experimental condition ,  those Pre-miRNA and mature miRNA  are expressed were considered. We selected those RBPs which have binding site on those miRNAs expressed in that particular experimental condition, also we included those RBPs and genes regulating  RBPs having binding sites on those miRNAs. Bayesian network is a type of probabilistic graphical model that represents a set of variables and their conditional dependencies via a directed acyclic graph (DAG). The basic model is a p-dimensional random vector X with joint distribution P.   Refer to the main idea of miRNA biogenesis, a more general approach is adopted via structural equation modelling (SEM) . in this approach , each conditional probability distribution(CPD) is directly modeled via a generalized linear model. The basic steps followed are structure learning for possible DAGs using the expression data, sparse regularized SEM  based on the CPD and decision on a suitable penalty  to model high dimensional data (n is very small as compare to p) are presented in fig. 4.  
       </div>
       <div class="col" style=" border: 3px solid green;text-align:justify"><br>
          <br><img class= "center" id="myImg_4" src="img/image4.jpeg" alt="expression of rbp" style="width:100%;max-width:600px; border:solid;"><br>  
     <!-- The Modal -->
	<div id="myModal_4" class="modal">
  <span class="close">&times;</span>
  <img class="modal-content" id="img04">
  <div id="caption"></div>
</div>

<script>
// Get the modal
var modal = document.getElementById('myModal_4');

// Get the image and insert it inside the modal - use its "alt" text as a caption
var img = document.getElementById('myImg_4');
var modalImg = document.getElementById("img04");
var captionText = document.getElementById("caption");
img.onclick = function(){
  modal.style.display = "block";
  modalImg.src = this.src;
  captionText.innerHTML = this.alt;
}

// Get the <span> element that closes the modal
var span = document.getElementsByClassName("close")[3];

// When the user clicks on <span> (x), close the modal
span.onclick = function() { 
  modal.style.display = "none";
}
</script>
          The algorithm followed in BNA are performed using “Sparsebn”, ccdr Algorithm and SparsebnUtils R-package and for more in detail kindly refer  to our article.From the result of BNA  miRNA-RBP regulomes  are collected i.e . RBPs responsible for processing of Pre-miRNA from Pri-miRNA and mature miRNA from Pre-miRNA along with the regulon of each RBPs associated in miRNA processing  in  each experimental condition. Further a unsupervised K-means clustering techniques has been implemented for  classification of disease and normal samples based on the expression of RBPs , miRNA and combination of both. At first the mature miRNAs are separated from the miRNA-RBP regulomes obtained from the BNA in different experimental conditions .  Also the RBPs  are extracted, that are involved in processing of that particular mature miRNA in each experimental condition. Again the expression of RBPs and miRNAs  are extracted from both disease and normal  conditions. Based on the expression data of RBPs or miRNAs or combination of both the samples are clustered implementing the K-means clustering algorithm.
    </div>   
      
    </div>
    
  </div>
</div>
</div>

  <!-- Accordion card -->

  <!-- Accordion card -->
  <div class="card">

    <!-- Card header -->
    <div class="card-header" role="tab" id="headingFour4">
      <a class="collapsed" data-toggle="collapse" data-parent="#accordionEx" href="#collapseFour4"
        aria-expanded="false" aria-controls="collapseFour4">
        <h5 class="mb-0">
         RESULT AND DISCUSSION <i class="fa fa-angle-down rotate-icon"></i>
        </h5>
      </a>
    </div>

    <!-- Card body -->
    <div id="collapseFour4" class="collapse" role="tabpanel" aria-labelledby="headingFour4" data-parent="#accordionEx">
    <div class="card-body">
    <div class="container-fluid">
    <div class="row">
    <div class="col" style=" border: 3px solid green;text-align: justify"><br>
    In human cells, most miRNAs are transcribed by RNAP II as  Pri-miRNAs  (Bartel,2004), which range from a few hundred nucleotides to several kilobases (kb). Some miRNAs are transcribed by their own promoter but the majority are found within introns (Bartel & Chen,2004). In our study  1kb flanking region around Pre-miRNA had considered, which we assume as putative  Pri-miRNA due to limited resources  regarding availability of  Pri-miRNAs . To find out the binding site of RBPs on  Pri-miRNA bed files of  46 RBPs  which included 222 CLIP-seq  samples were  intersects with  Pri-miRNAs and Pre-miRNAs using the Bed-tools intersect and the data collected from GEO for rest of 109 RBPs  which included 1008 CLIP-seq  samples were mapped on both Pri-miRNAs and Pre-miRNAs using Bowtie2. The detail statistics of binding sites of each RBP on different Pri-miRNA and Pre-miRNA are provided in the supplementary Table-4 and detail of number of RBPs bind in each Pri-miRNA and Pre-miRNA are provided in Table-5. From the result we found only 495 Pri-miRNAs have DROSHA  binding sites,where as DGCR8 have binding site on 1879 Pri-miRNAs.  Though we know that DROSHA is involved for processing of Pre-miRNA form Pri-miRNA , but it has been noticed that less number of  Pri-miRNAs have binding sites of DROSHA . To check the status of DROSHA in Pre-miRNA processing from Pri-miRNA further a correlation analysis has been performed based on the expression data of Pre-miRNA and DROSHA in our data which include 47 experimental conditions. From the correlation analysis we found 1376 Pre-miRNAs has  significant positive correlation with DROSHA, which  indicates that DROSHA have been involved in Pre-miRNA  processing from Pri-miRNA. From both approaches  it has been noticed that there are 361 Pri-miRNAs  are common, after discarding 361 Pri-miRNA from 495 Pri-miRNA we left with 134 Pri-miRNA. These Pri-miRNAs had included and  a total of 1410 out of 1881   Pri-miRNAs have DROSHA binding sites has been calculated. Similarly for DICER1 have binding sites on  only 189 Pre-miRNAs .  To check the status of DICER1we had   carried out the correlation analysis for DICER1 with  Pre-miRNAs and mature miRNAs considering the expression data of 47 experimental conditions.  those  Pre-miRNAs were considered  in which the DICER1 has a negative correlation and  mature miRNA generated from those Pre-miRNAs have positive correlation with DICER1. A total of 1290 Pre-miRNAs has negative correlation and their respective mature miRNAs has positive correlation with DICER1. Also it has been noticed that 163 Pre-miRNA shared both the data, so we found a total of 1316 Pre-miRNAs has DICER1 binding sites after excluding 163 Pre-miRNAs from one set of data . Most of   Pri-miRNAs and Pre-miRNAs  have binding sites for multiple RBPs , which indicated the involvement of multiple RBPs in miRNAs biogenesis. Also it had noticed that in some miRNAs there is no evidence of  binding  site  for  DROSHA/DICER, which create the possibility for involvement other RBPs besides DROSHA/DICER in miRNA biogenesis. From the result of binding site of RBPs on miRNAs (both Pri-miRNA & Pre-miRNA) ,the binding sites are clustered based on RBPs , Pre-miRNAs and Pri-miRNAs. 71   RBPs out of 155 RBPs have binding site on more than 1800 Pri-miRNAs ; 5 RBPs have binding site on more than 1700  Pri-miRNAs; 6 RBPs have binding site on more than 1600  Pri-miRNAs; 7 RBPs have binding sites on more than 1500 Pri-miRNAs; 3 RBPs have binding sites on more than 1400 Pri-miRNAs; 3 RBPs have binding sites on more than 1300 Pri-miRNAs; 4 RBPs have binding sites on more than 1200 Pri-miRNAs; 4 RBPs have binding sites on more than 1100 Pri-miRNAs; 4 RBPs have binding sites on more than 1000 Pri-miRNAs; where as only one  RBP have binding sites on more than 900 Pri-miRNAs; 4 RBPs have binding sites on more than 800 Pri-miRNAs; 2 RBPs have binding sites on more than 700 Pri-miRNAs;only one  RBP have binding sites on more than 600 Pri-miRNAs; 3 RBPs have binding sites on more than 500 Pri-miRNAs and rest 34 RBPs have binding site on less than 500 primary miRNAs. But in Pre-miRNA, 70 RBPs have binding site on less than 100 Pre-miRNAs;  8  RBPs have binding site on more than 100 Pre-miRNAs; 9 RBPs have binding site on more than 200 Pre-miRNAs;  6 RBPs have binding site on more than 300 Pre-miRNAs;  7 RBPs have binding site on more than 400 Pre-miRNAs;  4 RBPs have binding site on more than 500 Pre-miRNAs;  8 RBPs have binding site on more than 200 Pre-miRNAs; 5 RBPs have binding site on more than 700 Pre-miRNAs;   4 RBPs have binding site on more than 800 Pre-miRNAs; 7 RBPs have binding site on more than 200 Pre-miRNAs and the rest  27   RBPs have binding site on  more than 1000 Pre-miRNAs. the detail distribution of RBPs ( in percentage) binding sites on different  Pri-miRNA and Pre-miRNA are presented in fig.6. Also  binding site for  each RBP on both primary and Pre-miRNAs are provided in supplementary Table-4.1.
    </div>
    <div class="col" style="border: 3px solid green;text-align: justify"><br>
     Again  it has been noticed that  each Pri-miRNA have binding site of more than 70 RBPs, which indicate  involvement of  several RBPs besides DGCR8/DROSHA for processing of Pre-miRNAs from Primary miRNAs. Out of 1881 primary miRNAs , 88 Pri-miRNAs have binding site of more than 130 RBPs ; 909 Pri-miRNAs have binding site of more than 110 RBPs; 741 Pri-miRNAs have binding site of  more than 90 RBPs and rest 104 Pri-miRNAs have binding site of more than 70 RBPs. While in case of Pre-miRNA, out of 1881 Pre-miRNAs only 23 Pre-miRNAs have binding site of  more than 90 RBPs ; 46 Pre-miRNAs have binding site of more than 70 RBPs ; 275 Pre-miRNAs have binding sites of  more than 50 RBPs; 717 Pre-miRNAs have binding sites of more than 30 RBPs; 787 Pre-miRNAs have binding site of more than 10 RBPs and rest 32 Pre-miRNAs  have binding sites for less than 10 RBPs. the detail distribution of   percentage of miRNAs for different  RBPs presented in fig-7, . From the study it had detected that more number of RBPs bound in Pri-miRNA as compare to Pre-miRNA, but when normalized over length Pre-miRNAs have more number of  RBPs binding site as compare to Pri-miRNA . Gerstberger et al. (2014) and Loughlin et al. (2009) also reported that “most of RBPs preferred secondary structure of RNA-sequence which leads to strong positional inter-dependencies between different bases due to base-pairing”. To check significance of binding site of RBP over Pri-miRNA and Pre-miRNA, a Fisher’s exact test has been carried out  for each miRNA, which found to be highly significant and conclude Pre-miRNAs shared more binding site of different RBPs as compared to Pri-miRNA. the detail of normalized and original value of binding sites of  number of RBPs on each Pri-miRNA and   Pre-miRNA, also P-values of fisher's exact test for each miRNA  are provided in table -6. A very important observation about RBPs is that they follow regulon model of regulation where genes disposing towards some common functions have been found commonly regulated by some common RBP (Keene, 2007). In this study the regulatory chains for each RBP has been considered, initially possible partners of each RBPs are collected from string database which was a total of 1169 included both proteins and RBPs.  But in our study we have CLIP-seq data for only 155 RBPs ,which enable binding site information for those RBPs on different miRNAs. besides these 155 RBPs ,1169 RBPs and genes are partnered with these RBPs in regulation of gene expression and have no CLIP-seq data. In our study we have the expression data  of different RBPs,on the basis of expression data it is possible to draw a conclusion whether a RBP  bound or not.  To  understand the association of binding site of RBP with their expression ,  CLIP-seq data and RNA-seq data of 73 RBPs  belonging to same experimental conditions were collected . The CLIP-seq data were processed by following the protocol described in the material and method section . The processed CLIP-seq data reads were mapped on genome using Bowtie to find out the possible binding sites of  RBPs on human genome (hg38). Again to find out the expression of RBP in particular condition, RNA-seq expression analysis  carried out following the protocol described in the material and method section. Total binding sites of RBP in each sample for that particular experimental condition counted . A correlation analysis has been performed between the binding sites and expression of that RBP  for each experimental condition and the detail work flow has been presented in fig.-8 The detail  description of CLIP-seq and RNA-seq data are provided in the supplementary Table-3  . Here for strong association,   a correlation  greater than or equal to 0.8 had considered. In our study 60  RBPs (82%) out of 73 RBPs have positive correlation and is  more than 0.8 in all experimental conditions. Which indicate expression has a positive association with binding sites  of  RBP i.e. if expression of RBP increased then binding site of that RBP also increased which is  also reported by  Nicholson et al. (2017).    In  rest 13 RBPs it has been noticed that  correlation is   greater than  0.8 for some conditions and in other it is  less than 0.8.   To find out  the reason behind their depicted behavior in both conditions we had performed a co-expression network analysis, which will enable us to find out possible factors of that particular RBP responsible for its differences in correlation between its expression and binding sites.  Before performing co-expression network analysis, at first all possible partners of that particular RBP from the String database are included in the network along with their expression for that particular experimental condition.  An in house R-script has been prepared using the "vsn",“igraph” and "genefilter" R-packages for co-expression network analysis based on  expression data of RBP and its partners. In co-expression network analysis a absolute correlation greater than or equal to  0.8 has been considered for network construction.
     </div> 
     <div class="col" style=" border: 3px solid green;text-align:justify"><br>
    From the remaining thirteen RBPs in nine RBPs (Acin1, HNRNPA1, HNRNPF, HNRNPU, MBNL2, TRA2B, IGF2BP2, FUS and CTCF) the expression of that particular RBP is high  in some conditions and in other conditions its expression is very low. The correlation between expression and binding sites had correlation greater than 0.8 in those conditions where expression was high  and correlation less than 0.8 when expression was low. To check the significance of RBP expression in different conditions, a t-test had carried out and its found to be significant. So from our previous inference for binding site , we can conclude that  RBPs   having strong correlation between its expression and binding sites when its expression is high and vise-versa. For other four RBPs (AGO2, HNRNPA2B1, METTL3 and NSUN2)  it has been noticed that there was no significant difference in expression of RBP in both conditions but still there is a difference between correlation i.e. in some condition it is greater than 0.8 and in others it was less than 0.8. In such situations there  are possibilities for involvement of other factors (RBPs or proteins) with synergistic or antagonistic association with RBP which may help or preclude in binding to RNA. To find out the possible interaction between different RBPs or proteins , co-expression network analysis has carried out for those four RBPs. In co-expression  network spherical shapes are represent the RBPs and circular shapes represent the proteins. Green color edges are for positive correlation and red color edges are for negative correlation between two nodes. Edge length are scaled in the network based on  correlation i.e. for strong correlation the edge length is less as compared to a weak correlation. Also size of vertices's are scaled based on the expression value of RBPs or proteins in the network i.e. if a RBP or protein is highly expressed then their vertices's size are larger as compare to low expressed RBP or protein. Different RBPs or proteins co-expressed are belong to same module and have same color. For AGO2 RNA-seq  and CLIP-seq data were collected for two different experimental conditions. In these experimental conditions   duplicate RNA-Seq experiments for cells grown in the presence of modified nucleotides or in normal medium. It has been  observed that in normal condition the correlation of binding site with expression is not significant where as in treated condition it is significant. From the network we found that TNRC6A (which is also a RBP) play  a crucial role in binding of  AGO2 . Also for binding of AGO2 there is also another factor i.e. LSM6 played a crucial role . During binding of AGO2 ,  LSM6  also co-expressed and  positively regulate the AGO2. The favorable condition for AGO2 to bind  when TNRC6A has a positive association with LSM6 and LSM6 also have a positive association with AGO2.The detail network presented in fig-8.RNA-seq and CLIP-seq  experiment data are collected from GEO for HNRNPA2B1. The experiment  consist of two experimental condition and the detail of data presented in Table-3. the two experimental conditions are  to understand how RNA binding proteins control the splicing code  to human biology and disease. They  have presented a comprehensive study to elucidate how heterogeneous nuclear ribonucleo particle (hnRNP) proteins, among the most abundant RNA binding proteins, coordinate to regulate alternative Pre-mRNA splicing in human cells. from the correlation analysis of binding site and expression of RBP and it has strong  positive correlation in one condition and  weak correlation in another condition.  from the co-expression network analysis , it has been analyzed that ELAVL1 has play the vital role in binding of  HNRNPA1B2 in treated condition, it positively regulate HNRNPA2B1 and help in binding  in treated condition. Where as in control condition ELAVL1 has no significant positive association with HNRNPA2B1.The detail network  for HNRNPA2B1 regulation presented in fig .9 . Again for METTL3 both RNA-seq and CLIP-seq data were collected from  GEO database. Wild type MDA-MB-231 cells were subjected to the HITS-CLIP procedure obtained from the nuclear fraction . For RNA-seq, nuclear RNA was extracted from MDA-MB-231  and  Hela cells . The data consist of two experimental condition i.e. in MDA-MB-231 cells and Hella cells .  In  Hella cells  the correlation is found to be significant between expression and binding site of METTL3. But in MDA-MB-231 cells  there is no significant difference between expression of METTL3 with Hella cells, still it has  no significant correlation with binding site. To investigate this situation , a co-expression network analysis has been performed . From the network it has been noticed that  GTF2H1 a positive association with METTL3  which help in its binding ,  i.e whenever expression of GTF2H1 increases , it positively regulate METTL3 binding to the RNA.  The detail network has been presented in fig-10.
 </div>
</div>
 </div>
   
</div>
   

  </div>
  <!-- Accordion card -->

  <!-- Accordion card -->
  <div class="card">

    <!-- Card header -->
    <div class="card-header" role="tab" id="headingTwo2">
      <a class="collapsed" data-toggle="collapse" data-parent="#accordionEx" href="#collapseTwo2"
        aria-expanded="false" aria-controls="collapseTwo2">
        <h5 class="mb-0">
        CONCLUSION<i class="fa fa-angle-down rotate-icon"></i>
        </h5>
      </a>
    </div>

    <!-- Card body -->
    <div id="collapseTwo2" class="collapse" role="tabpanel" aria-labelledby="headingTwo2" data-parent="#accordionEx">
      <div class="card-body">
        ######
      </div>
    </div>

  </div>

</div>
<!-- Accordion wrapper -->
<!-- Image Showcases -->
    <section class="showcase">
      <div class="container-fluid p-0">
        <div class="row no-gutters">

          <div class="col-lg-6 order-lg-2 text-white showcase-img" style="background-image: url('img/biomolecules.png');"></div>
          <div class="col-lg-6 order-lg-1 my-auto showcase-text">
            <h2>About website</h2>
            <p class="lead mb-0"><strong>RBP-miRNA Crosstalking</strong> web-portal is a first of the dynamic portal which provide information about gene-regulation information achieved by interaction or crosstalking between RBP, precursor and mature miRNA. This portal take tissue or condition specific RBP and miRNA. This generate network from it.</p>
          </div>
        </div>
        <div class="row no-gutters">
          <div class="col-lg-6 text-white showcase-img" style="background-image: url('img/Language.jpg');"></div>
          <div class="col-lg-6 my-auto showcase-text">
            <h2>Technology Used</h2>
            <p>The portal is implemented using advance libraries of HTML5. Portal homepage and background pages are developed in HTML5/CSS3, JQuery/JavaScript with support of Bootstrap packages. The website provides the facility for gaining insight . All the interaction can be visualized in selective mode. Common target analysis is also activated for miRNAs which can be extended to the network. All the interaction can be visualized in selective mode.  All the network elements can be selected for pathways and functional enrichment analysis. The analysis can be done in comparative mode. The network relationships between nodes and edges are structured into JSON file via neomodel module of python comprising django webframework. A set of full network JSON file is loaded into D3, a web based visualization open library. Query sent by a user from HTML pages is processed through PHP in the background and retrieves results of respective sections </p>
          </div>
        </div>
        <div class="row no-gutters">
          <div class="col-lg-6 order-lg-2 text-white showcase-img" style="background-image: url('img/graph.png');"></div>
          <div class="col-lg-6 order-lg-1 my-auto showcase-text">
            <h2>Network generated &amp; Customize</h2>
            <p class="lead mb-0">Dynamic network generated at real time and customize the network by query based specific information generated by the user using Cypher query implimented in neomodal specific PYTHON driver for NEO4J Database.</p>
          </div>
        </div>
      </div>
    </section>

    <!-- Testimonials -->
    <section class="testimonials text-center bg-light" id="team-page">
      <div class="container">
        <h2 class="mb-5">Our Team...</h2>
        <div class="row">
          <div class="col-lg-4">
            <div class="testimonial-item mx-auto mb-5 mb-lg-0">
              <img class="img-fluid rounded-circle mb-3" src="Upendra.jpg" alt="Upendra">
              <h5>Upendra Kumar</h5>
              <p class="font-weight-light mb-0">Researcher and Data Analyst</p>
            </div>
          </div>
          <div class="col-lg-4">
            <div class="testimonial-item mx-auto mb-5 mb-lg-0">
              <img class="img-fluid rounded-circle mb-3" src="Ashwani.jpg" alt="Ashwani">
              <h5>Dr. Ashwani Kumar</h5>
              <p class="font-weight-light mb-0">WebDesigner and PDF fellow</p>
            </div>
          </div>
          <div class="col-lg-4">
            <div class="testimonial-item mx-auto mb-5 mb-lg-0">
              <img class="img-fluid rounded-circle mb-3" src="ravishankar.jpg" alt="ravishankar">
              <h5>Dr. Ravi Shankar</h5>
              <p class="font-weight-light mb-0">Principal Investigator</p>
            </div>
          </div>
        </div>
      </div>
    </section>

    <!-- Call to Action -->
    <section class="call-to-action text-white text-center">
      <div class="overlay"></div>
      <div class="container">
        <div class="row">
          <div class="col-xl-9 mx-auto">
            <h2 class="mb-4"></h2>
          </div>
        </div>
      </div>
    </section>
    

    <!-- Footer -->
    <footer class="footer bg-light">
      <div class="container">
        <div class="row">
          <div class="col-lg-6 h-100 text-center text-lg-left my-auto">
            <ul class="list-inline mb-2">
              <li class="list-inline-item">
                <a href="http://scbb.ihbt.res.in/">About</a>
              </li>
              <li class="list-inline-item">&sdot;</li>
              <li class="list-inline-item">
                <a href="contact.html">Contact</a>
              </li>
             </ul>
            <p class="text-muted small mb-4 mb-lg-0">&copy; SCBB,2019. All Rights Reserved.</p>
            <p class="text-muted small mb-4 mb-lg-0">&copy; Website is developed and maintained by Ashwani Kumar.</p> &nbsp;&nbsp;&nbsp;<p align="right" class="text-muted small mb-4 mb-lg-0">For any query mail us at ashwanibitian@gmail.com</p>
          
          </div>
    </footer>

    <!-- Bootstrap core JavaScript -->
    <script src="vendor/jquery/jquery.min.js"></script>
    <script src="vendor/bootstrap/js/bootstrap.bundle.min.js"></script>
    <script src="vendor/js/search.js"></script>

    <script>$(document).on('click', '[data-toggle="lightbox"]', function(event) {
                event.preventDefault();
                $(this).ekkoLightbox();
            }); </script>

  </body>

</html>
